### PR TITLE
Scripts to update asset with additional asset data

### PR DESCRIPTION
**Pull request by humulla**
There is a request to add the asset characteristics to the MMH property page. However, the data we currently have in Dynamodb is unreliable, hence we have first created a script to get the PK id with the property reference given and then to update the asset characteristics with the new data given. We are still expecting some new values to be provided to us but I can confirm that the script works as expected in dev.

### CODE REVIEW

File: use_cases.py [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
-def amend_tenure():
-    amend_tenure_main()
+# def amend_tenure():
+#     amend_tenure_main()
+    
+# def get_assets_by_prop_ref():
+#     get_assets_by_prop_ref_main()
+
+# def patch_asset_with_additional_data():
+#     patch_asset_with_additional_data_main()
```
Comment (Duslerke) [inline use_cases.py:36]: Should this be committed?
These are just function definitions, not the function calls themselves.

Comment (humulla) [reply use_cases.py:36]: Yeah I know, but if you don't comment it out python script acts weird and runs them, found it best to comment it all out and only have the one you need!

Comment (adamtry) [reply use_cases.py:36]: Thinking about it I think the imports should be left commented for easy access to uncomment and run them, and the functions in the middle should be removed.
In hindsight they're somewhat redundant which is my mistake - you can just call the functions from the imports directly in the `__main__` clause.

File: aws/src/database/dynamodb/scripts/asset_table/get_assets_by_prop_ref.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+def main():
+    table = get_dynamodb_table(Config.TABLE_NAME, Config.STAGE)
+    _file_path = "aws\src\database\data\input\ProdAdditionalAssetData.csv"
+    asset_csv_data = csv_to_dict_list(_file_path)
+
+    # Note: Writing to TSV which can be imported into Google Sheets
+    asset_with_ids = set_id_in_csv_with_asset_id(table, asset_csv_data)
+
+    with open("assets.tsv", "w") as f:
```
Comment (Asura5) [inline aws/src/database/dynamodb/scripts/asset_table/get_assets_by_prop_ref.py:79]: if you're reading from a csv, is this a typo?

**Review by adamtry — APPROVED** [review]

**Review by adamtry — APPROVED** [review]

**Pull request merged**